storybook/src/components/form/form-group-date-picker.css-template.ts

```diff
@@ -8,32 +8,48 @@ import { ComponentImplementation } from "../../templates";
 export const cssFormGroupDatePicker: ComponentImplementation<FormGroupDatePicker<TemplateResult>> = {
   component: "formGroupDatePicker",
   implementation: "html-css",
-  template: ({ datePickerTemplate }) =>
-    function formGroupDatePickerTemplate({ id, state, required, helpText, errorText, datePicker }) {
+  template: ({ datePickerTemplate, infoButtonTemplate, infoTemplate }) =>
+    function formGroupDatePickerTemplate({
+      id,
+      state,
+      required,
+      helpText,
+      errorText,
+      datePicker,
+      label,
+      info,
+      infoButton,
+    }) {
       const errorTextId = `${id}-error-text`;
       const helpTextId = `${id}-help-text`;
+      const infoTextId = `${id}-info-text`;
 
       const ariaDescribedBy = helpText ? helpTextId : undefined;
 
       const ariaErrorMessage = errorText ? errorTextId : undefined;
 
       return html`
-        <fieldset
+        <div
           class="form-group dso-input dso-input-date ${classMap({
             "dso-required": !!required,
             [`dso-${state}`]: !!state,
           })}"
           aria-describedby=${ifDefined(ariaDescribedBy)}
           aria-errormessage=${ifDefined(ariaErrorMessage)}
         >
+          <div class="dso-label-container">
+            <label for=${id} class="control-label"> ${label} </label>
```

Comment:
De spaties mogen weg.

En waar verwijst het `id` attribuut naar? Dat lijkt mij te moeten verwijzen naar het input veld in `<dso-date-picker>`, maar ik zie niet voor mij hoe dat nu zou werken.

Reply:
Hmm goed gezien, volgens mij moet ik dit nog toevoegen `${datePickerTemplate({...datePicker, id: this.id} ?? {})}`

Reply:
Ik denk `${datePickerTemplate({...datePicker, id})}`, een leeg object is nooit nullish waardoor `?? {}` nooit wordt toegepast.